<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>517</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&amp;Search:</string>
            </property>
            <property name="buddy">
             <cstring>leSearch</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSearch">
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewPeople"/>
    <addaction name="actionViewRelations"/>
    <addaction name="actionViewPlaces"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddItem"/>
    <addaction name="actionEditItem"/>
    <addaction name="actionDeleteItem"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonFollowStyle</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionViewPeople"/>
   <addaction name="actionViewRelations"/>
   <addaction name="actionViewPlaces"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/opened_folder-32.png</normaloff>:/icons/opened_folder-32.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open an existing database</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/save_as-32.png</normaloff>:/icons/save_as-32.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save the current database under a different name</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/exit-32.png</normaloff>:/icons/exit-32.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit the application</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/about-32.png</normaloff>:/icons/about-32.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About this application</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/add_file-32.png</normaloff>:/icons/add_file-32.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create a new database</string>
   </property>
   <property name="priority">
    <enum>QAction::HighPriority</enum>
   </property>
  </action>
  <action name="actionViewPeople">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/user-32.png</normaloff>:/icons/user-32.png</iconset>
   </property>
   <property name="text">
    <string>People</string>
   </property>
   <property name="toolTip">
    <string>View the list of people</string>
   </property>
  </action>
  <action name="actionViewPlaces">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/world_map-32.png</normaloff>:/icons/world_map-32.png</iconset>
   </property>
   <property name="text">
    <string>Places</string>
   </property>
   <property name="toolTip">
    <string>View the list of places</string>
   </property>
  </action>
  <action name="actionAddItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/Add_Row-32.png</normaloff>:/icons/Add_Row-32.png</iconset>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
   <property name="toolTip">
    <string>Adds a new item to the list</string>
   </property>
  </action>
  <action name="actionEditItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/edit-32.png</normaloff>:/icons/edit-32.png</iconset>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
   <property name="toolTip">
    <string>Edits the currently selected item</string>
   </property>
  </action>
  <action name="actionDeleteItem">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/Delete_Row-32.png</normaloff>:/icons/Delete_Row-32.png</iconset>
   </property>
   <property name="text">
    <string>Delete...</string>
   </property>
   <property name="toolTip">
    <string>Deletes the currently selected item</string>
   </property>
  </action>
  <action name="actionViewRelations">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icons/wedding_rings-32.png</normaloff>:/icons/wedding_rings-32.png</iconset>
   </property>
   <property name="text">
    <string>Relations</string>
   </property>
   <property name="toolTip">
    <string>View the list of relations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
